### PR TITLE
RA-1848:Unbundle spa-module in refApp2.11.0

### PR DESCRIPTION
**Pull request by sherrif10**
https://issues.openmrs.org/browse/RA-1848
cc @dkayiwa  @mozzy11

### CODE REVIEW

File: package/src/main/resources/openmrs-distro.properties [hunk 1]
```diff
@@ -40,7 +40,6 @@ omod.adminui=${adminuiVersion}
 omod.reportingcompatibility=${reportingcompatibilityVersion}
 omod.legacyui=${legacyuiVersion}
 omod.fhir2=${fhirVersion}
-omod.owa=${owaVersion}
```
Comment (mozzy11) [inline package/src/main/resources/openmrs-distro.properties:43]: @sherrif10 , did have to remove this ??

**Comment by dkayiwa** [comment]
@sherrif10 some travis failure to feast on.

**Comment by sherrif10** [comment]
@dkayiwa, @mozzy Sure working on it asap

File: package/src/main/resources/openmrs-distro.properties (hunk 1 above)
Comment (sherrif10) [reply package/src/main/resources/openmrs-distro.properties:43]: Resolved

**Comment by sherrif10** [comment]
It functions perfect after testing, so its now fully unbundled  cc @dkayiwa  @mozzy11

**Comment by mozzy11** [comment]
Thanks @sherrif10

**Comment by sherrif10** [comment]
Thanks @mozzy11

**Pull request merged**